src/components/MAPI/clusters/utils.ts

```diff
@@ -317,6 +321,7 @@ function createDefaultAWSCluster(config: {
         [capiv1beta1.labelClusterName]: config.name,
         [infrav1alpha3.labelOrganization]: config.organization,
         [infrav1alpha3.labelReleaseVersion]: config.releaseVersion,
+        [capiv1beta1.labelServicePriority]: config.servicePriority,
```

Comment:
We only want to set the service priority label on the main cluster (`cluster.clusterx-k8s.io`) resource. Can we avoid setting this label on the provider cluster (`azurecluster.infrastructure.cluster.x-k8s.io` and `awscluster.infrastructure.giantswarm.io`) resources?